diff --git a/js/back_to_top.js b/js/back_to_top.js
index 0f6000b..0ff9fd2 100644
--- a/js/back_to_top.js
+++ b/js/back_to_top.js
@@ -18,7 +18,7 @@
           $("html, body").bind("scroll mousedown DOMMouseScroll mousewheel keyup", function () {
             $('html, body').stop();
           });
-          $('html,body').animate({scrollTop: 0}, 1200, 'easeOutQuart', function () {
+          $('html,body').animate({scrollTop: 0}, 300, 'linear', function () {
             $("html, body").unbind("scroll mousedown DOMMouseScroll mousewheel keyup");
           });
           return false;
